src/external/wannier_shared.jl

```diff
@@ -1,5 +1,85 @@
+# Shared functionality to operate with Wannier programs (Wannier.jl and Wannier90).
+# The communication happens with the Wannier90 file formats which Wannier.jl also understands.
+
 using Dates
 
+"""
+A Gaussian-shaped initial guess. Can be used as an approximation of an s- or σ-like orbital.
+"""
+struct GaussianWannierProjection
+    center::AbstractVector{Float64}
+end
+
+function (proj::GaussianWannierProjection)(basis::PlaneWaveBasis, qs)
+    # associate a center with the fourier transform of the corresponding gaussian
+    # TODO: what is the normalization here?
+
+    model = basis.model
+    map(qs) do q
+        q_cart = recip_vector_red_to_cart(model, q)
+        exp( 2π*(-im*dot(q, proj.center) - dot(q_cart, q_cart) / 4) )
+    end
+end
+
+"""
+A p-like initial guess, using the difference of two Gaussians with different centers.
+"""
+function opposite_gaussians_projection(center1::AbstractVector, center2::AbstractVector)
```

Comment:
I'm not too sure myself. If you already know that you want a plike orbital you probably want to use `HydrogenicWannierProjection` anyway (or even the pswfc from the PSP in the future).

Reply:
yes, that's true. Good point. OTOH this makes it questionable why this function even exists in the main code. Might be good to include in the example to show how custom projectors could be made. Maybe it's better put there.

Reply:
Just removed it. I don't think that encouraging custom projectors is particularly a good idea, in principle it shouldn't be needed and the interface is still experimental. (I was using `opposite_gaussians_projection` before the hydrogenic guess was implemented, now it can be useful for comparison against the hydrogenic guess but that doesn't need to be in the mainline code).

Reply:
>  (I was using opposite_gaussians_projection before the hydrogenic guess was implemented, now it can be useful for comparison against the hydrogenic guess but that doesn't need to be in the mainline code).

Yes, but it would be good to keep the idea documented in the wannier example. That's what I meant. Sorry if I was not clear.